### PR TITLE
AUI-2106 Update HISTORY.md

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-scheduler/js/aui-scheduler-base.js [hunk 1]
```diff
@@ -1447,6 +1466,19 @@ var SchedulerBase = A.Component.create({
...
+        },
+
+        /**
+         * Set the `showToolbar` property - i.e., defines if the scheduler
+         * header will be displayed.
+         *
+         * @method _uiSetShowToolbar
+         * @protected
+         */
+        _uiSetShowToolbar: function() {
```
Comment (thiago-rocha) [inline src/aui-scheduler/js/aui-scheduler-base.js]: This seems weird, a private function that only calls a public function, could you explain the why you had to create this function?

File: src/aui-scheduler/js/aui-scheduler-base.js [hunk 2]
```diff
@@ -774,6 +775,17 @@ var SchedulerBase = A.Component.create({
         },
 
         /**
+         * Define whether the scheduler toolbar will be displayed.
+         *
+         * @attribute showToolbar
```
Comment (thiago-rocha) [inline src/aui-scheduler/js/aui-scheduler-base.js]: Could you add `@default` on this attribute?

File: src/aui-scheduler/HISTORY.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 ## @VERSION@
 
+* [AUI-2106](https://issues.liferay.com/browse/AUI-2106) It should be possible to not show the scheduler's toobar.
```
Comment (thiago-rocha) [inline src/aui-scheduler/HISTORY.md]: `typo` ...scheduler's toolbar.

File: src/aui-scheduler/js/aui-scheduler-base.js (hunk 1 above)
Comment (brandizzi) [inline src/aui-scheduler/js/aui-scheduler-base.js]: To be fair, the called function should be protected as well, but it was not created that way and I'd rather not change it.

**Comment by brandizzi** [comment]
Thanks for the remarks. You can take a look at the pull request now.

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Sending it again with last adjustments: . Later I'll proceed with extracting the toolbar from the scheduler.

**Pull request closed without merging**